### ISSUE TITLE
Not an issue - feature request - Linux support?

### ISSUE THREAD

**Issue by calumcraig** [uffou/MixCloud-Play#2]
Do you have any plans to support this on Linux? As it is an Electron app, it is possible - I was actually able to build it and get it running on Linux but of course it didn't integrate with the OS. (I came across this looking for a desktop MixCloud player that I can control from the taskbar as I can other media players.)

**Comment by uffou** [comment]
@calumcraig yes, this could easily be ported to Linux. I will add support if I have some time next month.

**Comment by calumcraig** [comment]
That would be great, I look forward to it.
